core/os/os.h

```diff
@@ -285,7 +285,8 @@ class OS {
 	String get_safe_dir_name(const String &p_dir_name, bool p_allow_paths = false) const;
 	virtual String get_godot_dir_name() const;
 
-	virtual String get_data_path() const;
+	virtual String get_data_home() const;
```

Comment:
This change breaks compatibility pretty seriously and I don't think it's needed, I'd just leave it as is

Reply:
Sure, this was just to align with the xdg spec.

If we keep this as `get_data_path`, I'm a bit worried that `get_data_dirs` will be confusing, because it doesn't actually include the 'home' path.

I think my best idea for a name is `get_data_search_paths`, and I think it would be best to include `get_data_path()` in the list.  So you have `get_data_path` for a single, writable data dir, and `get_data_search_paths` when you want to search for something that may be in a non-writable location.

Also, just for my curiosity, are the compatibility concerns because of the script interface, or the C++ interface?  I assume we could alias the script function if we had to?

Reply:
I've done mostly as I described and removed any renaming.